core/blockchain.go

```diff
@@ -1726,6 +1749,7 @@ func (bc *BlockChain) insertChain(chain types.Blocks, setHead bool) (int, error)
 		if err != nil {
 			return it.index, err
 		}
+		statedb.SetLogger(bc.logger)
 
 		// Enable prefetching to pull in trie node paths while processing transactions
 		statedb.StartPrefetcher("chain")
```

Comment:
It appears the prefetcher is configured with a live tracer using this config which leads to concurrent tracing which is incorrect.

Before starting the prefetcher at https://github.com/ethereum/go-ethereum/pull/27629/files#diff-53d5f4b8a536ec2a8c8c92bf70b8268f1d77ad77e9f316e6f68a2bcae5303215R1766, we need to change the `Tracer` to be `nil`, here the change I've applied locally:

```
				go func(start time.Time, followup *types.Block, throwaway *state.StateDB) {
					config := bc.vmConfig
					config.Tracer = nil

					bc.prefetcher.Prefetch(followup, throwaway, config, &followupInterrupt)
					...
```

Reply:
Oh right, thanks for the catch. Fixed.